spec/append.test.sh

```diff
@@ -49,10 +49,8 @@ argv.py "${y[@]}"
 s='abc'
 s+=(d e f)
 echo $s
```

Comment:
Hm s is an array  now, so I think we should also test `argv.py "${s[@]}"`

Reply:
Thanks. I updated the test in commit c5fd118c7.